### PR TITLE
Revert "Automate test case for 'wego add --url' flag"

### PR DESCRIPTION
**Pull request by rokshana-b**
Reverts weaveworks/weave-gitops#301

Changes in this PR are causing acceptance suite failure

### CODE REVIEW

**Comment by rokshana-b** [comment]
Changes in this PR are causing acceptance suite failure

**Review by J-Thompson12 — APPROVED** [review]

File: test/acceptance/test/add_tests.go [hunk 1]
```diff
@@ -279,18 +279,18 @@ var _ = Describe("Weave GitOps Add Tests", func() {
...
-		branchName := "test-branch"
+		branchName := "main"
 		appManifestFilePath := "./data/nginx.yaml"
 		defaultSshKeyPath := os.Getenv("HOME") + "/.ssh/id_rsa"
 		appRepoName := "wego-test-app-" + RandString(8)
 		workloadName := "nginx"
 		workloadNamespace := "my-nginx"
-		url := "ssh://git@github.com/" + os.Getenv("GITHUB_ORG") + "/" + appRepoName + ".git"
-		addCommand := "app add . --url=" + url + " --branch=" + branchName + " --dry-run"
+		url := "ssh://git@github.com/weaveworks-gitops-test/" + appRepoName + ".git"
```
Comment (josecordaz) [inline test/acceptance/test/add_tests.go:292]: @rokshana-b Would it be ok to use the env var instead?. It looks like you are reverting a previous change right? Something happened?

Comment (josecordaz) [reply test/acceptance/test/add_tests.go:292]: I just saw the description of this PR 🙂

Comment (rokshana-b) [reply test/acceptance/test/add_tests.go:292]: My tests passed with `SmokeTests` tag, but the acceptance suite failed as a whole when I removed the tags. Something went very wrong... I'm reverting all changes

**Pull request merged**